components/template-mgt/org.wso2.carbon.identity.template.mgt/src/main/java/org/wso2/carbon/identity/template/mgt/TemplateMgtConstants.java

```diff
@@ -41,6 +46,30 @@
     public static final String STATUS_INTERNAL_SERVER_ERROR_MESSAGE_DEFAULT = "Internal server error";
 
     public static final String TEMPLATE_RESOURCE_PATH = "/";
+    public static final Path SP_TEMPLATES_DIR_PATH = Paths.get(CarbonUtils.getCarbonHome(), "repository",
+            "resources", "identity", "application-templates");
```

Comment:
rather than having the directory paths as repository/resources/identity/application-templates, IMO it would be more clear if we have the following structure. 

repository/resources/identity/templates/applications
repository/resources/identity/templates/identity-providers
repository/resources/identity/templates/emails etc.

WDYT? @AnuradhaSK @thanujalk

Reply:
addressed in feace15